README.md

```diff
@@ -10,32 +10,275 @@
+    * Add student with id A0123456A to module CS2103
+* `add student m/CS2100 i/A0123457A n/Amy t/@amylee e/amy@u.nus.edu`:
+    * Add student with id A0123457A to module CS2100
+
+### Edit a student's information: `edit student`
+
+Edit a student's information.<br>
+The identity fields(module name/student id) must be provided.<br>
+At least one optional editable field (name/tele handle/email) must be provided, and only provided editable fields will be changed.
+Student name should only contain alphanumeric characters and spaces, and it should not be blank. <br>
+Student tele handle must start with @, and followed by 5-32 characters containing only alphanumeric characters and the underscore '_'.<br>
+Student email must follow the format specified in the [Parameter summary](#parameter-summary-with-prefix-included) section below.<br>
+If duplicate fields are provided, only the latest fields will be processed.
+
+Format: `edit student m/<module name> i/<student id> [n/<edited student name>] [t/<edited student tele handle>] [e/<edited student email>]`
+
+Examples:
+* `edit student m/CS2103 i/A0123456A n/John Doe`:
+    * Edit the name of the student with id A0123456A in module CS2103 to John Doe
+* `edit student m/CS2100 i/A0123457A t/@amylee e/amy@u.nus.edu`:
+    * Edit the tele handle and email of the student with id A0123457A in module CS2100 to @amylee and amy@u.nus.edu respectively
+
+### Delete a student from a specific module: `delete student`
+
+Delete the specified student from the specified module.
+
+Format: `delete student m/<module name> i/<student id>`
+
+Examples:
+* `delete student m/CS2103 i/A1234567A`:
+    * Delete student with id A1234567A from module CS2103’s list of students
+* `delete student m/CS2100 i/A7654321A`:
+    * Delete student with id A7654321A from module CS2100’s list of students
+
+### Add a new task to a module: `add task`
+
+Add a new task to the specified module.<br>
+Task id must begin with a capital letter 'T', followed by numbers only.<br>
+Task name must only contain alphanumeric characters and spaces, and it must not be blank.<br>
+Task deadline must contain either date only/time only/date and time.<br>
+Date must be in the form "YYYY-MM-DD".<br>
+Time must be in the form "hh:mm" in 24-hrs format, and will appear in TAB in 12-hrs format.
+
+Format: `add task m/<module name> ti/<task id> a/<task name> d/<task deadline>`
+
+Examples:
+* `add task m/CS2103 ti/T1 a/assignment1 d/2021-10-12`:
+    * Add the task assignment1 with deadline 'Tue, Oct 12 2021' to module CS2103
+* `add task m/CS2100 ti/T1 a/lab1 d/2021-10-13 16:00 `:
+    * Add the task lab1 with deadline 'Wed, Oct 13 2021 04:00 pm' to module CS2100
+
+### Edit a task's information: `edit task`
+
+Edit a task's information.<br>
+The identity fields (module name/task id) must be provided.<br>
+At least one optional editable field (name/deadline) must be provided, and only provided editable fields will be changed.<br>
```

Comment:
same as the student one, I think it would be better to include that it is possible to edit the task even if you provide the same name/deadline

Reply:
agreed